weedcoco/index/indexing.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         self.es_index_name = es_index_name
         self.es_type_name = es_type_name
         self.batch_size = batch_size
-        self.es_client = Elasticsearch(HOST=es_host, PORT=es_port)
+        self.es_client = Elasticsearch([{"host": es_host, "port": es_port}])
```

Comment:
This line appears to have broken the continuous integration, or at least the process is failing in this class, with:
```pytb
 File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/Weed-ID-Interchange/Weed-ID-Interchange/weedcoco/index/indexing.py", line 149, in <module>
    es_index.post_to_index()
  File "/home/runner/work/Weed-ID-Interchange/Weed-ID-Interchange/weedcoco/index/indexing.py", line 135, in post_to_index
    helpers.bulk(self.es_client, index_batch)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/helpers/actions.py", line 390, in bulk
    for ok, item in streaming_bulk(client, actions, *args, **kwargs):
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/helpers/actions.py", line 320, in streaming_bulk
    **kwargs
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/helpers/actions.py", line 247, in _process_bulk_chunk
    for item in gen:
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/helpers/actions.py", line 196, in _process_bulk_chunk_error
    raise error
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/helpers/actions.py", line 235, in _process_bulk_chunk
    resp = client.bulk("\n".join(bulk_actions) + "\n", *args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/client/utils.py", line 152, in _wrapped
    return func(*args, params=params, headers=headers, **kwargs)
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/client/__init__.py", line 460, in bulk
    body=body,
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/transport.py", line 390, in perform_request
    raise e
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/transport.py", line 365, in perform_request
    timeout=timeout,
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/elasticsearch/connection/http_urllib3.py", line 258, in perform_request
    raise ConnectionError("N/A", str(e), e)
elasticsearch.exceptions.ConnectionError: ConnectionError(<urllib3.connection.HTTPConnection object at 0x7f72723bf590>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution) caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f72723bf590>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/urllib3/connection.py", line 170, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/urllib3/util/connection.py", line 73, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
```